packages/devextreme/js/ui/html_editor/modules/mentions.js

```diff
@@ -313,6 +313,10 @@ if(Quill) {
                 itemTemplate,
                 searchExpr
             });
+
+            const $container = this._list._$container;
+            this._list.option('focusedElement', getPublicElement($container));
+            this.quill.root.setAttribute('aria-activedescendant', $container.attr('id'));
```

Comment:
i dont believe this is a plce to go, we can set active descendant right after we rendered the list. ( by getting id from list.getFocusedItemId() for example )